patches/capitalize-app-id.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/assets/freedesktop/org.zealdocs.zeal.appdata.xml.in b/assets/freedesktop/org.zealdocs.zeal.appdata.xml.in
+index 88abd54..a74ff91 100644
+--- a/assets/freedesktop/org.zealdocs.zeal.appdata.xml.in
++++ b/assets/freedesktop/org.zealdocs.zeal.appdata.xml.in
+@@ -1,6 +1,6 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+ <component type="desktop">
+-  <id>org.zealdocs.zeal</id>
++  <id>org.zealdocs.Zeal</id>
```

Comment:
I changed this in https://github.com/zealdocs/zeal/commit/d4d7d35c3e137804d583c1b33c2c0116d796fbb6 because of some warning that AppStream specification prefers lower-case id. Not really sure what is the best way.

Reply:
I see... I think we'd have to ask Flathub admins to make this repository lowercase too, and I'm not sure if that would break things elsewhere.

So I'm honestly not sure if it's worth it going through all that, especially considering that this patch seems enough to fix things.